scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -213,9 +230,15 @@ function Initialize-IoTEdge {
         throw
     }
 
+    if (-not (Test-VcRuntimePresent)) {
```

Comment:
Some of these changes are not in master.

Reply:
Ah, I was wondering why they looked familiar. You've included changes from https://github.com/Azure/iotedge/pull/1871 We need to merge that one first.

Reply:
ugh! sorry, I thought all my PRs were in and my fork master was completely up to date.

Reply:
ok, so 1871 is in now.  can we complete this?

Reply:
Cherry-pick 0f46e9956e06e6ca4b1f3a76bdb71cc1e73f7b1e instead of the individual commits (esp since the PR got more commits than you have here when it was merged).

```sh
git reset --hard cdb6b950d65a7223c0df7f0f320f4e7a6b28ad0d
git cherry-pick 0f46e9956e06e6ca4b1f3a76bdb71cc1e73f7b1e
git push -f
```